src/components/global/Footer.vue

```diff
@@ -2,7 +2,8 @@
 	v-footer(padless dark)
 		v-card(class="flex" flat tile)
 			v-card-title(class="teal")
-				h4 VogonWeb #[strong Beta]
+				h4 VogonWeb #[strong Beta]&nbsp;
+				small {{version}}
 				
 				a.links(href="https://github.com/diging/vogon-web/releases/tag") Version 0.4
```

Comment:
lets add the version here like it was before instead of where you currently have it. Also for some reason these buttons get rendered in the DOM but do not display so even though that's not part of the ticket lets take care of that as well. If you look in the inspector you will see that these button get rendered and you can click on them if you can find them but they do not display on the page.